modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPInstanceLocalServiceImpl.java

```diff
@@ -665,7 +670,7 @@ public void deleteCPInstances(long cpDefinitionId) throws PortalException {
 			cpInstancePersistence.findByCPDefinitionId(cpDefinitionId);
 
 		for (CPInstance cpInstance : cpInstances) {
-			cpInstanceLocalService.deleteCPInstance(cpInstance);
+			cpInstanceLocalService.deleteCPInstance(cpInstance, false);
```

Comment:
I would expect to have the parameter makeCopy in this method and call pass false in the deleteCPDefinition also for SKUs

Reply:
Also becasue this method could be called elsewhere